src/main/java/uk/gov/di/ipv/domain/ErrorResponse.java

```diff
@@ -11,7 +11,11 @@
     MissingAuthorisationCode(1003, "Missing authorization code for token request"),
     FailedToExchangeAuthorizationCode(1004, "Failed to exchange the authorization code for an access token"),
     MissingAccessToken(1005, "Missing access token from user info request"),
-    FailedToParseAccessToken(1006, "Failed to parse access token");
+    FailedToParseAccessToken(1006, "Failed to parse access token"),
+    MissingAuthorizationCode(1007, "Missing authorization code"),
```

Comment:
There is already a MissingAuthorisationCode one up above on line 11. Looks like spelt with an 's' though, your spelling is probably the better one since that's what we use elsewhere. We probably don't need both here though?

Reply:
I've pushed a change to use the enum with id 1003, but changed the spelling to align with the OAuth RFC language. Removed the dupe enum.